src/java/com/vaklinov/zcashui/ZCashClientCaller.java

```diff
@@ -6,7 +6,7 @@
  * /____\____\__,_|___/_| |_|____/ \_/\_/ |_|_| |_|\__, | \_/\_/ \__,_|_|_|\___|\__|\___/|___|
  *                                                 |___/
  *
- * Copyright (c) 2016 Ivan Vaklinov <ivan@vaklinov.com>
+ * Copyright (c) 2017-2018 The Hush Developers <contact@myhush.org>
```

Comment:
Instead of removing the copyright line, it's is more appropriate and correct to add a brand new line, leaving
the old copyright years and names alone. So just keep your current Copyright line and revive the line that was deleted. For example, here is a copyright lines of the src/rpcmining.cpp file from hush.git:
```
src/rpcmining.cpp
1:// Copyright (c) 2010 Satoshi Nakamoto
2-// Copyright (c) 2009-2014 The Bitcoin Core developers
3-// Copyright (c) 2017      The Hush developers
```

Please fix this each file, since we want to give credit where credit is due.